### PR TITLE
Update `CONTRIBUTORS.md` and fix TYPO

### PR DESCRIPTION
**Pull request by rotcivegaf**
## Motivation

I participated in the code4rena audit contest
And fix a little typo in README.md

## Solution

**README.md:**
```diff
- You may wish to include a `.env` file that `export`s a specific profile when developing locally.
+ You may wish to include a `.env` file that `export` a specific profile when developing locally.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ FOUNDRY_PROFILE=lite forge test # with 1000 fuzz runs
 FOUNDRY_PROFILE=local-ffi forge test # compiles and deploys ReferenceConsideration normally, with 1000 fuzz runs
 ```
 
-You may wish to include a `.env` file that `export`s a specific profile when developing locally.
+You may wish to include a `.env` file that `exports` a specific profile when developing locally.
```
[the remark this answers is not in the document]
Comment (rotcivegaf) [reply README.md:116]: yes, it's true, tanks!

**Comment by rotcivegaf** [comment]
Move to #380

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ FOUNDRY_PROFILE=lite forge test # with 1000 fuzz runs
 FOUNDRY_PROFILE=local-ffi forge test # compiles and deploys ReferenceConsideration normally, with 1000 fuzz runs
 ```
 
-You may wish to include a `.env` file that `export`s a specific profile when developing locally.
+You may wish to include a `.env` file that `export` a specific profile when developing locally.
```
Comment (0age) [inline README.md:116]: how about "to `export` a specific profile"

**Comment by 0age** [comment]
Closing this for now as I don't think we've substantially improved on documentation; feel free to reopen with an improvement!

**Pull request closed without merging**